repo_tests/install_error_paths_are_reachable_15825_test.py

```
@@ -0,0 +1,339 @@
+case "$*" in
+  *auth/login*)          echo '{{"access_token":"t"}}' ;;
+  *api/nodes*)           printf '201' ;;
+  *code-source/assign*)  printf '200' ;;
+  *)                     printf '200' ;;
+esac
+exit 0
+"""
+
+
+@pytest.mark.parametrize(
+    "fail_glob,expected_warning",
```

Ruff (0.16.3)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)